assets/src/parse_bundle_plugin.js

```diff
@@ -34,11 +34,11 @@ var parseBundlePlugin = function(data, base_dir) {
     // plugins (including the core app).
     var bundle = _.clone(base_config);
     var library;
-    if ((typeof data.entry_file !== "undefined") &&
+    if ((typeof data.src_file !== "undefined") &&
         (typeof data.name !== "undefined") &&
-        (typeof data.module_path !== "undefined") &&
+        (typeof data.static_dir !== "undefined") &&
         (typeof data.stats_file !== "undefined")) {
-        bundle_data[data.name] = path.join(data.module_path, data.entry_file);
+        bundle_data[data.name] = data.src_file;
```

Comment:
I tried my best to write JS, but I think that after this PR, if you feel like it, perhaps see if there are more things that can be moved over to the Python app (the webpack_json command) to deliver trusted output that the webpack builder doesn't have to process.